### ISSUE TITLE
Expanding the Combat tab to include Spells/Prayers?

### ISSUE THREAD

**Issue by ChapterGrim** [fmasa/wfrp-master#259]
A way, probably manually, to set up a Spell/Prayer in the Combat tab would save a lot of faffing about, and jumping around between different sources for information. It be nice to have all the spellcasting info in a single tab, but that'd be a ludicrous amount of work I think, and there'd be way more flexible utility to a tool in the Combat tab that makes it easy to drop in a spellcasting or prayer with CN versus Channelled SLs, and Casting SLs plus Magic Missile boxes for manual inputs.

Honestly, this concept could also be used for a quick tool for calculating other combat scenarios or for random equipment used as weapons etc, stuff that maybe isn't in their trappings beyond a particular scenario? Or even just to calculate accumulated SLs at all, 
given that quite often multiple Talents consistently apply. I'd love to be able to build a quick reference for my Gnome Wizard's Stealth Tests using Channelling Ulgu, so perhaps a "Tests" quick reference tab is worthwhile?

If you could input individual Tests like custom Skills or Talents in that tab, with the facility to add titles for variables, like Talents, Conditions, things effecting the Winds of Magic etc, you could "build" a Test that you commonly have to make with everything you need in one place. It might make sense having this in the Combat tab honestly, as this essentially expands the utility of that tab...

**Comment by fmasa** [comment]
> A way, probably manually, to set up a Spell/Prayer in the Combat tab would save a lot of faffing about, and jumping around between different sources for information. It be nice to have all the spellcasting info in a single tab, but that'd be a ludicrous amount of work I think, and there'd be way more flexible utility to a tool in the Combat tab that makes it easy to drop in a spellcasting or prayer with CN versus Channelled SLs, and Casting SLs plus Magic Missile boxes for manual inputs.

Yeah, I'm not very happy about the current state of Combat tab, I agree that for Priests and or Wizards it's not very useful. I will try to design something and post some ideas 👍

> Honestly, this concept could also be used for a quick tool for calculating other combat scenarios or for random equipment used as weapons etc, stuff that maybe isn't in their trappings beyond a particular scenario? Or even just to calculate accumulated SLs at all,
given that quite often multiple Talents consistently apply. I'd love to be able to build a quick reference for my Gnome Wizard's Stealth Tests using Channelling Ulgu, so perhaps a "Tests" quick reference tab is worthwhile?

I think this falls under https://github.com/fmasa/wfrp-master/issues/72? I'm still scoping that, so feel free to drop by and post some of these suggestions there 🙂
